reckless_cmd/src/reckless/mod.rs

```diff
@@ -46,9 +48,21 @@ impl PluginManager for RecklessManager {
         Ok(())
     }
 
-    async fn install(&mut self, plugins: &[&str]) -> Result<(), RecklessError> {
-        // FIXME: Fix debug message with the list of plugins to be installed
-        debug!("installing plugins");
+    async fn install(&mut self, plugins: &Vec<String>) -> Result<(), RecklessError> {
+        debug!("installing plugins {:?}", plugins);
+        for plugin_to_be_installed in plugins {
+            for repo in &self.repos {
+                for plugin_list in repo.list().await {
```

Comment:
```suggestion
                if let Some(plugin_in_repo) = repo.get_plugin_by_name(plugin_name) {
                      // please avoid unwrapping on configure this is the worst this to do! 
                      // in 50% of the case this method fails, and this will crash reckless
                       let path = plugin_in_repo.configure().await.unwrap();
                       debug!("{path}");
                       continue;
                } 
                // return a not found plugin error
```

You had the power to change the API so we can make the code a little bit cleaner